openapi-specs/sdwan/unified/Applications.yaml

```diff
@@ -4339,7 +4339,7 @@ components:
     ListQueryResponse:
       properties:
         count:
-          description: 'The actual count.
+          description: 'The count of the query list.
```

Comment:
What's this?

This whole directory sdwan/unified isn't ours. Did you modify this by accident?